api/src/main/java/org/pmiops/workbench/api/DataSetController.java

```diff
@@ -609,6 +624,32 @@ private void formatTimestampValues(List<DataSetPreviewValueList> valuePreviewLis
     return ResponseEntity.ok(dataSetResponse);
   }
 
+  @Override
+  public ResponseEntity<DataDictionaryEntry> getDataDictionaryEntry(
+      Long cdrVersionId, String domain, String domainValue) {
```

Comment:
Enum instead of string for `domain`?

Reply:
I was unable to use object types in the parameter if I wanted to keep these as query parameters. I think it is possible if I change it to be a request body parameter but it seemed wrong to change the look of the API because of a swagger limitation.

FYI - OpenAPI 3.0 supports this. One more reason to upgrade.

I did end up changing the query parameters to path parameters because that felt more correct.

Reply:
Unfortunate, thanks - sounds good.